UNSIGNED_SWEEP.md

```diff
@@ -0,0 +1,43 @@
+# Unsigned Sweep Recovery Instructions
+
+Build an unsigned transaction using the user and backup public keys, independent of any BitGo services. You should use this for Cold Wallet recoveries.
+
+## What happens in an unsigned sweep recovery?
+
+1. You are asked to provide the Coin Name that you want to recover.
+
+2. You are asked to provide the Environment (Testnet/Mainnet).
+
+3. You are asked to provide the User Public Key as found on your recovery KeyCard.
+
+4. You can provide User Key ID as provided on your KeyCard. Most wallets will not have this, and you can leave it blank
+
+5. You are asked to provide the Backup Public Key as found on your recovery KeyCard.
+
+6. You can provide Backup Key ID as provided on your KeyCard. Most wallets will not have this, and you can leave it blank
+
+7. You are asked to provide the BitGo Public Key as found on your recovery KeyCard.
+   
+8. You are asked to provide the Destination Address (The address your recovery transaction will send to).
+
+9. For UTXO based coins like Bitcoin, Litecoin, Dash, Zcash, Bitcoin Gold, Tron, Bitcoin Cash, Bitcoin SV, Bitcoin ABC (eCash), Address Scanning Factor needs to be provided. This is the count of consecutive addresses without transaction to scan before stopping the tool. By default, it's value is 20 and it can be kept the same.
+
+10. For Coins like Stellar, Ripple, and EOS, Root Address of the wallet need to be provided.
+
+11. For Ethereum and ERC20 Token, Wallet Contract Address need to be provided. This is the Eth address of the wallet contract. This is also the wallet's base address.
+
+12. For ERC20 Token, Token Contract Address need to be provided. This is the address of the smart contract of the token to recover. This is unique to each token and is NOT your wallet address
+
+13. For Ethereum and ERC20 Token, Gas Price in Gwei need to be provided. The value should be between 1 Gwei and 2500 Gwei. The default is 20 Gwei.
+
+14. For Ethereum and ERC20 Token, Gas Limit need to be provided. The value should be between 30,000 and 20,000,000. The default is 500,000 units of gas.
+
+15. An API Key token from etherscan.com need to be provided for Ethereum recoveries.
+
+16. The SDK determines the amount of Coin that can be recovered, and builds an unsigned transaction to send that amount to the Destination Address.
+
+17. You need to half sign this unsigned transaction with your user private key using Offline Vault Console.
+
+18. You need to fully sign the half signed transaction with your backup private key using Offline Vault Console.
+
+19. This completed transaction is a valid transaction and can be broadcasted on the network to finish the recovery.
```

Comment:
How does the user broadcast it to the network? Let's explain it here.

Reply:
Added along with link to block explorers